website/content/v1.2/reference/cli.md

```diff
@@ -2191,11 +2197,13 @@ talosctl upgrade [flags]
 ### Options
 
 ```
+      --debug          debug operation from kernel logs. --no-wait is set to false when this flag is set
   -f, --force          force the upgrade (skip checks on etcd health and members, might lead to data loss)
   -h, --help           help for upgrade
   -i, --image string   the container image to use for performing the install
   -p, --preserve       preserve data
   -s, --stage          stage the upgrade to perform it after a reboot
+      --wait           wait for the operation to complete, tracking its progress. always set to true when --debug is set
```

Comment:
the gif shows the `--no-wait` flag, and we have the `--wait` flag

Reply:
oh good one, yes, video is old now. I'll re-record

Reply:
just re-recorded it